hello-books-swagger.json

```diff
@@ -290,336 +349,49 @@
-            "required": true,
-            "type": "integer"
-          },
-          {
-            "in": "body",
-            "name": "body",
-            "description": "This is the request body object containing books' information",
-            "required": true,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "verification": {
-                  "type": "object",
-                  "example": "verified"
-                }
-              }
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Book Verification status successfully changed"
-          },
-          "400": {
-            "description": "Incorrectly filled field"
-          },
-          "401": {
-            "description": "Unauthorized access by a non-admin"
-          },
-          "404": {
-            "description": "Book to be verified does not exist"
-          },
-          "500": {
-            "description": "Internal Server Error"
-          }
-        },
-        "security": [
-          {
-            "Bearer": []
-          }
-        ]
-      },
-      "get": {
-        "tags": ["Books"],
-        "summary": "Get a Book in The Library",
-        "description": "This endpoint enables users fetch a book the library",
-        "consumes": ["application/json"],
-        "produces": ["application/json"],
-        "parameters": [
-          {
-            "name": "id",
-            "in": "path",
-            "description": "ID of book",
-            "required": true,
-            "type": "integer"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Book Successfully Returned",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "status": {
-                  "type": "string",
-                  "example": "success"
-                },
-                "data": {
-                  "type": "object",
-                  "example": {
-                    "id": 2,
-                    "title": "Antman & The Wasp",
-                    "body": "Antman is back!",
-                    "description": "Scott Lang teams up with the Wasp to save the world again",
-                    "pages": "70",
-                    "verification": "pending",
-                    "createdOn": "2019-06-19T11:48:57.366Z",
-                    "hardcopy": true,
-                    "status": "available",
-                    "uploadId": null,
-                    "authorName": "Stan Lee",
-                    "genre": "Fantasy",
-                    "uploader": "kyloRen"
-                  }
-                }
-              }
-            }
-          },
-          "404": {
-            "description": "Book not found"
-          },
-          "500": {
-            "description": "Internal Server Error"
-          }
-        }
-      },
-      "delete": {
```

Comment:
Great work overall @abdulfatai360, I especially like that you did some house cleaning on the codebase. Thanks for that!

Looking at this file though, it seems you have deleted the existing Swagger documentation for all books while you were pulling the develop branch into you, please is that something you could look into? Thank you